ipc/vmpart/blockchainGateway.go

```diff
@@ -472,6 +473,26 @@ func (blockchain *BlockchainHookGateway) ExecuteSmartContractCallOnOtherVM(_ *vm
 	return nil, errors.New("not implemented")
 }
 
+// RoundTime not used in 1.2
+func (blockchain *BlockchainHookGateway) RoundTime() uint64 {
```

Comment:
I do not think this is needed, as we have 2 blockchain hook interfaces, one is legacy and one is the new one. Can we check if this works without these changes ?